src/ray/test/run_core_worker_tests.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
we can pass `args` to bazel `cc_test`. Then we don't need this extra script.

Reply:
That sounds doable if we move the redis stuff to the test itself.
On the other side, it seems all the existing integration tests in ray are using this kind of scripts.

Reply:
I think it'd be preferable to only use `cc_test`. Because all test code is consolidated in one place. 
However, if it's hard move redis to C++ test. It's okay to use scripts as well.

Reply:
Probably can leave this as is for now, later we can revisit together w/ other integration tests and update them together if necessary.

Reply:
ok. Then should we add this script to CI?